src/opensynth/models/faraday/gaussian_mixture/gmm_init.py

```diff
@@ -14,23 +14,23 @@ def initialise_gmm_params(
     reg_covar: float = 1e-6,
 ) -> gmm_model.GMMInitParams:
     """
-    Initialise Gaussian Mixture Parameters. This works
-    by only initialising on the first batch of the data
-    using K-means, and porting SKLearn's implementation
-    of computing cholesky precision and covariances.
+    Initialise Gaussian Mixture Parameters.
+    Component means and component assignments are initialised using K-means,
+      and porting SKLearn's implementation of computing cholesky precision and
+      covariances.
 
     Args:
-        X (np.array): Input data
+        X (torch.Tensor): Input data
```

Comment:
Eh - the Args are not correct! They all form part of the doc strings!

Reply:
Updated